src/components/molecules/modals/ShopFinishedModal.js

```diff
@@ -0,0 +1,206 @@
+  // </Column>
+)
+
+const SuccessMessage = ({ order }) => {
+  console.log({ order })
+  return (
+    <Fragment>
+      <div className="row marginBottom">
+        <div className="column">
+          <Text className="bangers" fontSize="4rem" margin="0">
+            {order?.product.message}
+          </Text>{' '}
+        </div>
+      </div>
+      <div className="row ">
+        <div className="column  left">
+          <InfoColumn label={'Order ID'} value={order?._id} third />
+        </div>
+      </div>
+
+      <div className="row marginTop">
+        <Text className="bold" fontSize="1.3rem" margin="0 0 5px 0">
+          Order details
+        </Text>
+      </div>
+
+      <div className="row ">
+        <InfoColumn label={'Quantity'} value={order?.details.quantity} third />
+        <InfoColumn label={'Price'} value={`$ ${order?.details.price}`} third />
+        <InfoColumn label={'Total'} value={`$ ${order?.details.total}`} third />
+      </div>
+
+      {order?.product?.isPhysical && order?.details?.clientNote > '' ? (
+        <div className="row  marginBottom">
+          <InfoColumn
+            label={'Your message'}
+            value={order?.details.clientNote || '-'}
+            fullwidth
+            marginTop
+          />
+        </div>
+      ) : (
+        <div className="row marginTop marginBottom">
+          <div className="column">
+            <StyledButton
+              className="fullwidth"
+              active
+              onClick={() => window.open(order?.product.downloadLink, '_blank')}
+            >
+              {order?.product.downloadLabel || 'Download link'}
+            </StyledButton>
+          </div>
+        </div>
+      )}
+      <div className="row marginTop">
+        <div className="column">
+          <Text fontSize="1.1rem" className="center info" margin="0">
+            You will shortly receive your receipt by e-mail{' '}
+          </Text>
+        </div>
+      </div>
+    </Fragment>
+  )
+}
+
+export const ShopFinishedModal = ({
+  isOrderSuccess,
+  isPreviewMobile,
+  isSidePreview,
+  onLoadOrder,
+  onClose,
+  shopItem,
+  show,
+}) => {
+  const [ssrDone, setSsrDone] = useState(false)
+  const [orderAvailable, setOrderAvailable] = useState(true)
+
+  const initialOrder = {
+    client: {},
+    details: {
+      clientNote: 'Default niote',
+    },
+    product: {
+      name: 'Default name',
+      isPhysical: true,
+      downloadLink: '#',
+      downloadLabel: '#',
+      message: 'Default thanks!',
+    },
+  }
+  const [order, setOrder] = useState(initialOrder)
+
+  const handleClose = () => {
+    const si = document.location.href.indexOf('success=')
+    const newUrl = document.location.href.slice(0, si - 1)
+    history.replaceState(null, '', newUrl)
+    console.log({ newUrl })
+    onClose()
+  }
+
+  console.log({ shopItem })
+
+  useEffect(() => {
+    // setOrderAvailable(true)
+    if (isOrderSuccess) {
+      ;(async () => {
```

Comment:
Is this necessary or it's a typo?

Reply:
it's something that lint does. Technically it is required. Without it the code will not be interpreted correctly.